### PR TITLE
add update methods on users/profiles/roles

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

File: src/security/kuzzleSecurity.js [hunk 1]
```diff
@@ -163,6 +163,46 @@ KuzzleSecurity.prototype.createRole = function (id, content, options, cb) {
   }
 };
 
+
+/**
+ * Update a role in Kuzzle.
+ *
+ * @param {string} id - profile identifier
```
Comment (AnthonySendra) [inline src/security/kuzzleSecurity.js]: niptick: role identifier

File: src/security/kuzzleSecurity.js [hunk 2]
```diff
@@ -163,6 +163,46 @@ KuzzleSecurity.prototype.createRole = function (id, content, options, cb) {
...
+ * @param {object} content - a plain javascript object representing the profile's modification
```
Comment (AnthonySendra) [inline src/security/kuzzleSecurity.js]: niptick: role's modification

**Comment by AnthonySendra** [comment]
+1

**Comment by benoitvidis** [comment]
+1

**Pull request merged**